### PR TITLE
Update New-CsTeamsMessagingPolicy.md

### PR DESCRIPTION
**Pull request by angchin**
Adding new ChatPermissionRole policy

### CODE REVIEW

File: skype/skype-ps/skype/New-CsTeamsMessagingPolicy.md [hunk 1]
```diff
@@ -382,6 +382,16 @@ Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
+### -ChatPermissionRole
+Determines the Supervised Chat role of the user.  Set this to Full to allow the user to supervise chats.  Supervisors have the ability to initiate chats with and invite any user within the enviornment.  Set this to Limited to allow the user to initiate conversations with Full and Limited permissioned users, but not Restricted.  Set this to Restricted to block chat creation with anyone other than Full permissioned users.  
+
+```yaml
+Type: String
```
Comment (get-itips) [inline skype/skype-ps/skype/New-CsTeamsMessagingPolicy.md:389]:
```suggestion
Type: String
Parameter Sets: (All)
Aliases:

Required: False
```

**Review by dariomws — APPROVED** [review]
I'm ok once the @get-itips suggestion is applied. Thanks!

File: skype/skype-ps/skype/New-CsTeamsMessagingPolicy.md [hunk 2]
```diff
@@ -382,6 +382,16 @@ Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
+### -ChatPermissionRole
+Determines the Supervised Chat role of the user.  Set this to Full to allow the user to supervise chats.  Supervisors have the ability to initiate chats with and invite any user within the enviornment.  Set this to Limited to allow the user to initiate conversations with Full and Limited permissioned users, but not Restricted.  Set this to Restricted to block chat creation with anyone other than Full permissioned users.
```
Comment (dariomws) [inline skype/skype-ps/skype/New-CsTeamsMessagingPolicy.md:386]:
```suggestion
Determines the Supervised Chat role of the user.  Set this to Full to allow the user to supervise chats.  Supervisors have the ability to initiate chats with and invite any user within the environment.  Set this to Limited to allow the user to initiate conversations with Full and Limited permissioned users, but not Restricted.  Set this to Restricted to block chat creation with anyone other than Full permissioned users.  
```

Comment (dariomws) [reply skype/skype-ps/skype/New-CsTeamsMessagingPolicy.md:386]: Just another small fix.

**Pull request merged**